node-scripts/spec/pipeline3/scripts.spec.js

```diff
@@ -517,4 +414,92 @@ describe("Scripts", function() {
         "\n"
     );
   });
+
+  it("should generate 'replace environment variable' script", function() {
+    var envVar = "KEY";
+    var value = "env.value";
+    var filename = ".env";
+
+    expect(scripts.writeProperty(envVar, value, filename)).toEqual(
+      'if ! grep -R "^[#]*s*KEY.*" .env > /dev/null; then\n' +
+        "\techo \"APPENDING because 'KEY' not found\"\n" +
+        '\techo "KEY=env.value" >> .env\n' +
+        "else\n" +
+        "\techo \"SETTING because 'KEY' found already\"\n" +
+        '\tsed -i "s/^[#]*\\s*KEY.*/KEY=env.value/" .env\n' +
+        "fi\n"
+    );
+  });
+
+  it("should generate 'create distro.env file' script", function() {
```

Comment:
"should generate script to create environment file"

Reply:
Isn't there any `expect` in this test?

Reply:
`except` added